alvr/server/cpp/platform/linux/EncodePipelineVAAPI.cpp

```diff
@@ -42,7 +42,7 @@ void set_hwframe_ctx(AVCodecContext *ctx, AVBufferRef *hw_device_ctx)
   frames_ctx->sw_format = AV_PIX_FMT_NV12;
   frames_ctx->width = ctx->width;
   frames_ctx->height = ctx->height;
-  frames_ctx->initial_pool_size = 3;
+  frames_ctx->initial_pool_size = 1;
```

Comment:
Why change it to one?

Reply:
@nowrep can you please explain this change?

Reply:
I thought it's pool for the mapped frames (there is only one frame now). But in any case, it makes no difference since the pool will just resize when needed.